docs/concepts/models.md

```diff
@@ -11,20 +11,33 @@ Models share many similarities with Python's dataclasses, but have been designed
 differences that streamline certain workflows related to validation, serialization, and JSON schema generation.
 You can find more discussion of this in the [Dataclasses](dataclasses.md) section of the docs.
 
-Untrusted data can be passed to a model and, after parsing and validation, Pydantic guarantees that the fields
+Untrusted data can be passed to a model and, after validation, Pydantic guarantees that the fields
 of the resultant model instance will conform to the field types defined on the model.
 
 !!! note
-    Pydantic is primarily a parsing and transformation library, **not a validation library**.
-    Validation is a means to an end: building a model which conforms to the types and constraints provided.
+    The use of the terms "parse" and "validate" within Pydantic's context has been a source of confusion.
+    While these terms were previously used interchangeably, moving forward, we aim to exclusively employ "validate",
+    with "parse" reserved specifically for discussions related to [JSON parsing](../concepts/json.md).
 
-    In other words, Pydantic guarantees the types and constraints of the output model, not the input data.
+    The potential confusion arises from the fact that, strictly speaking, Pydantic's primary focus
```

Comment:
I think we want to say earlier on that "validation" is "the most widely understood colloquial term for the task pydantic performs" or similar.

Generally we should sound more like "this is the state of the world, we're not fighting it" rather than "we made a mistake long ago and we're sticking to that mistake.

Reply:
You could say something like "for brevity we use the term "validation" to refer to the process of optional coercion and validation which Pydantic is known for".?